src/lightning_app/core/queues.py

```diff
@@ -377,6 +408,13 @@ def length(self):
         val = self.client.get(f"/v1/{self.app_id}/{self._name_suffix}/length")
         return int(val.text)
 
+    def get_all(self, timeout: float):
+        time.sleep(timeout)
+        resp = self.client.post(f"v1/{self.app_id}/{self._name_suffix}", query_params={"action": "lrange"})
+        if resp.status_code == 204:
+            raise queue.Empty
+        return pickle.loads(resp.content)
```

Comment:
This won't work as the backend is going to respond with a json array.

Reply:
I can fix this once the backend implementation is done

Reply:
Yes, this is meant to be implemented by you ;)